.dockerignore

```diff
@@ -0,0 +1,12 @@
+.git
+
+.install
+
+tests
+contrib
+
+modules/rtm/src/RTM/*.o
+modules/rtm/src/RTM/*/*.o
+modules/rtm/src/RTM/*/*/*.o
+modules/rtm/src/*.mod
+modules/rtm/src/*.so
```

Comment:
Why did you move all of these files into `gh-pages`? All that should be there is the HTML required to host the website. I would delete all of these, since they unnecessarily bloat the website, and introduce confusion.